changelog/5657.improvement.md

```diff
@@ -0,0 +1 @@
+Add list handling in the `send_custom_json` method on `channels/facebook.py`.
```

Comment:
I think this changelog entry should actually be a `bugfix` rather than `improvement` based on the [original issue](https://github.com/RasaHQ/rasa/issues/5657). It would also be great if you could give more details around the bug since it isn't very clear for me in which circumstances would the `json_message` be of type `list` rather than `dict`.